src/coreclr/utilcode/longfilepathwrappers.cpp

```diff
@@ -697,10 +697,10 @@ HRESULT LongFile::NormalizePath(SString & path)
 
 BOOL LongFile::ContainsDirectorySeparator(SString & path)
 {
-    return path.Find(path.Begin(), DirectorySeparatorChar) || path.Find(path.Begin(), AltDirectorySeparatorChar);
+    return path.Find(path.Begin(), DirectorySeparatorChar);
```

Comment:
Should `AltDirectorySeparatorChar` handling stay here for Windows?

Reply:
This function is actually unused, I will remove it. However, the `IsDirectorySeparator` where I've also removed the alt directory handling needs to get it back, as it is used on the code path of the `LoadLibraryExWrapper` where the normalization of the path was added few years ago to handle issue with some NuGet packages containing paths with all forward slashes.

Reply:
I am not sure though with all the usages of the `NormalizePath` that calls the `IsDirectorySeparator`. All the wrappers in this file that get file path as an input call the `NormalizePath`. @jkotas do you see a reason for accepting both forms of slash in methods like `GetFileAttributesWrapper`, `CreateFileWrapper` etc? I have thought the `LoadLibraryExWrapper` is an exception.

Reply:
Do Windows APIs consider the slash and backslash to be directory separators? That would be surprising to me. I don't see that mentioned in the MSDN doc at https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file.

Reply:
Hmm, I've tried it and they really do support that. Interesting.